modules/networking/outputs.tf

```
@@ -1,15 +1,19 @@
 output "vpc_id" {
-  value = aws_vpc.default.id
+  value       = aws_vpc.default.id
+  description = "The ID of the VPC that hosts the cache and the executors in."
 }
 
 output "subnet_id" {
-  value = var.nat == true ? aws_subnet.private.0.id : aws_subnet.default.id
+  value       = var.nat == true ? aws_subnet.private.0.id : aws_subnet.default.id
+  description = "The subnet to run the VM in."
 }
 
 output "ip_cidr" {
-  value = local.ip_cidr
+  value       = local.ip_cidr
+  description = "The internal address that is owned by the subnetwork."
```

I don't fully understand that description, maybe something along these lines?

```suggestion
  description = "The CIDR range of the subnet in which the executor machines will be spawned."
```

---

I actually just took the docs from `ip_cidr_range` on [google_compute_subnetwork](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_subnetwork)

---

lol, good docs :D do you unterstand what they mean? maybe it's just me then.

---

I had to search online a bit to understand